diff -urN libgr-2.0.13/config.guess libgr-2.0.13-mark/config.guess
--- libgr-2.0.13/config.guess	1996-10-30 06:55:09.000000000 +0200
+++ libgr-2.0.13-mark/config.guess	2003-01-10 15:10:13.000000000 +0200
@@ -103,7 +103,7 @@
     CRAY-2:UNICOS:*:*)
 	echo cray2-cray-unicos
         exit 0 ;;
-    i[345]86:Linux:*:*)
+    i[3456]86:Linux:*:*)
 	echo ${UNAME_MACHINE}-unknown-linux
 	exit 0 ;;
     i[345]86:UNIX_SV:4.*:*)
diff -urN libgr-2.0.13/Makefile libgr-2.0.13-mark/Makefile
--- libgr-2.0.13/Makefile	1997-04-04 07:25:53.000000000 +0300
+++ libgr-2.0.13-mark/Makefile	2003-01-10 15:11:47.000000000 +0200
@@ -6,6 +6,7 @@
 # targets
 most: archive shared
 install : install_a install_s install_h
+	ldconfig
 everything: most progs
 install_everything : install install_p install_m
 
